ccl/streamingccl/streamingest: TestStreamIngestionFrontierProcessor failed

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3287579&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3287579&tab=artifacts#/) on master @ [eb1b0d71f948d6e5ba741c7747255f0bc48021d4](https://github.com/cockroachdb/cockroach/commits/eb1b0d71f948d6e5ba741c7747255f0bc48021d4):


```
I210811 07:40:52.565877 5550 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n3,pebble,s3] 3172  [JOB 286] compacted(elision-only) L6 [000413] (1.0 K) + L6 [] (0 B) -> L6 [] (0 B), in 0.0s, output rate 0 B/s
I210811 07:40:52.565991 5551 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3173  [JOB 286] sstable deleted 000413
I210811 07:40:52.566048 5549 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3174  [JOB 285] sstable deleted 000409
I210811 07:40:52.566084 5549 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3175  [JOB 285] sstable deleted 000410
I210811 07:40:52.566103 5549 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3176  [JOB 285] sstable deleted 000411
I210811 07:40:52.566123 5549 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3177  [JOB 285] sstable deleted 000412
I210811 07:40:52.566142 5549 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3178  [JOB 285] sstable deleted 000414
I210811 07:40:52.566160 5549 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3179  [JOB 285] sstable deleted 000417
I210811 07:40:52.567067 5634 kv/kvserver/raft_snapshot_queue.go:129  [n1,raftsnapshot,s1,r21/1:/Table/2{5-6}] 3180  skipping snapshot; replica is likely a LEARNER in the process of being added: (n3,s3):3LEARNER
I210811 07:40:52.567177 43 kv/kvserver/store_snapshot.go:1138  [n1,replicate,s1,r21/1:/Table/2{5-6}] 3181  streamed INITIAL snapshot 0e82cdca at applied index 19 to (n3,s3):3LEARNER in 0.00s @ 4.1 MiB/s: kv pairs: 12, log entries: 0, rate-limit: 8.0 MiB/s, queued: 0.00s
I210811 07:40:52.568636 5312 kv/kvserver/replica_raftstorage.go:824  [n3,s3,r21/3:{-}] 3182  applying snapshot of type INITIAL [id=0e82cdca index=19]
I210811 07:40:52.569134 5312 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3183  [JOB 287] ingesting: sstable created 000419
I210811 07:40:52.569198 5312 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3184  [JOB 287] ingesting: sstable created 000424
I210811 07:40:52.569237 5312 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3185  [JOB 287] ingesting: sstable created 000420
I210811 07:40:52.569263 5312 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3186  [JOB 287] ingesting: sstable created 000421
I210811 07:40:52.569288 5312 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3187  [JOB 287] ingesting: sstable created 000422
I210811 07:40:52.569310 5312 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3188  [JOB 287] ingesting: sstable created 000423
I210811 07:40:52.569379 5312 3@vendor/github.com/cockroachdb/pebble/event.go:617  [n3,pebble,s3] 3189  [JOB 288] WAL created 000425 (recycled 000395)
I210811 07:40:52.569578 5636 3@vendor/github.com/cockroachdb/pebble/event.go:585  [n3,pebble,s3] 3190  [JOB 289] flushing 1 memtable to L0
I210811 07:40:52.569635 5636 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3191  [JOB 289] flushing: sstable created 000426
I210811 07:40:52.569954 5636 3@vendor/github.com/cockroachdb/pebble/event.go:589  [n3,pebble,s3] 3192  [JOB 289] flushed 1 memtable to L0 [000426] (6.3 K), in 0.0s, output rate 19 M/s
I210811 07:40:52.570094 5637 3@vendor/github.com/cockroachdb/pebble/event.go:573  [n3,pebble,s3] 3193  [JOB 290] compacting(default) L0 [000426] (6.3 K) + L6 [000418] (239 K)
I210811 07:40:52.570518 5637 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3194  [JOB 290] compacting: sstable created 000427
I210811 07:40:52.573243 5312 3@vendor/github.com/cockroachdb/pebble/event.go:609  [n3,pebble,s3] 3195  [JOB 287] ingested L0:000419 (1.3 K), L0:000424 (1.2 K), L0:000420 (1.6 K), L0:000421 (1.0 K), L0:000422 (1.0 K), L6:000423 (1.0 K)
I210811 07:40:52.573728 5312 kv/kvserver/replica_raftstorage.go:845  [n3,s3,r21/3:/Table/2{5-6}] 3196  applied snapshot of type INITIAL [total=5msingestion=6@5msid=0e82cdca index=19]
I210811 07:40:52.574638 43 kv/kvserver/replica_command.go:2118  [n1,replicate,s1,r21/1:/Table/2{5-6}] 3197  change replicas (add [(n3,s3):3] remove []): existing descriptor r21:/Table/2{5-6} [(n1,s1):1, (n2,s2):2, (n3,s3):3LEARNER, next=4, gen=3]
I210811 07:40:52.578095 5530 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n2,pebble,s2] 3198  [JOB 324] compacted(default) L0 [000419 000424 000420 000421 000422] (5.9 K) + L6 [000427] (226 K) -> L6 [000428] (226 K), in 0.0s, output rate 8.3 M/s
I210811 07:40:52.580196 43 kv/kvserver/replica_raft.go:277  [n1,s1,r21/1:/Table/2{5-6}] 3199  proposing SIMPLE(v3) [(n3,s3):3]: after=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I210811 07:40:52.580783 5532 3@vendor/github.com/cockroachdb/pebble/event.go:573  [n2,pebble,s2] 3200  [JOB 325] compacting(elision-only) L6 [000423] (1.0 K) + L6 [] (0 B)
I210811 07:40:52.581029 5532 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n2,pebble,s2] 3201  [JOB 325] compacted(elision-only) L6 [000423] (1.0 K) + L6 [] (0 B) -> L6 [] (0 B), in 0.0s, output rate 0 B/s
I210811 07:40:52.581183 5578 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3202  [JOB 325] sstable deleted 000423
I210811 07:40:52.582621 5531 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3203  [JOB 324] sstable deleted 000419
I210811 07:40:52.582740 5531 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3204  [JOB 324] sstable deleted 000420
I210811 07:40:52.582788 5531 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3205  [JOB 324] sstable deleted 000421
I210811 07:40:52.582810 5531 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3206  [JOB 324] sstable deleted 000422
I210811 07:40:52.582830 5531 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3207  [JOB 324] sstable deleted 000424
I210811 07:40:52.582848 5531 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3208  [JOB 324] sstable deleted 000427
I210811 07:40:52.591786 5637 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n3,pebble,s3] 3209  [JOB 290] compacted(default) L0 [000426] (6.3 K) + L6 [000418] (239 K) -> L6 [000427] (242 K), in 0.0s, output rate 11 M/s
I210811 07:40:52.592139 5638 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3210  [JOB 290] sstable deleted 000418
I210811 07:40:52.592331 5638 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3211  [JOB 290] sstable deleted 000426
I210811 07:40:52.592567 5639 3@vendor/github.com/cockroachdb/pebble/event.go:573  [n3,pebble,s3] 3212  [JOB 292] compacting(default) L0 [000419 000424 000420 000421 000422] (6.1 K) + L6 [000427] (242 K)
I210811 07:40:52.593123 5639 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3213  [JOB 292] compacting: sstable created 000428
I210811 07:40:52.595052 43 testutils/testcluster/testcluster.go:1157  [-] 3214  WaitForFullReplication took: 3.019015885s
=== CONT  TestStreamIngestionFrontierProcessor
    stream_ingestion_frontier_processor_test.go:242: Leaked goroutine: goroutine 5737 [chan send]:
        github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest.(*mockStreamClient).ConsumePartition.func1(0xc002e32780, 0xc002e32e40, 0xc00494dd00, 0x1, 0x1, 0x5938510, 0xc00428b900, 0xc005b94cc0, 0x44dc461, 0x12)
        	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest/stream_ingestion_processor_test.go:95 +0x26d
        created by github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest.(*mockStreamClient).ConsumePartition
        	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest/stream_ingestion_processor_test.go:87 +0x15c
--- FAIL: TestStreamIngestionFrontierProcessor (8.97s)
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestStreamIngestionFrontierProcessor PKG=./pkg/ccl/streamingccl/streamingest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestStreamIngestionFrontierProcessor [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3294297&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3294297&tab=artifacts#/) on master @ [812ba837ed23a3a245faecd59700d82297abb4d4](https://github.com/cockroachdb/cockroach/commits/812ba837ed23a3a245faecd59700d82297abb4d4):


```
I210812 18:27:25.247650 5749 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3276  [JOB 353] sstable deleted 000428
I210812 18:27:25.247787 5749 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3277  [JOB 353] sstable deleted 000429
I210812 18:27:25.247873 5749 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3278  [JOB 353] sstable deleted 000430
I210812 18:27:25.247937 5749 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3279  [JOB 353] sstable deleted 000432
I210812 18:27:25.247998 5749 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3280  [JOB 353] sstable deleted 000434
I210812 18:27:25.248156 5752 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n2,pebble,s2] 3281  [JOB 355] compacted(elision-only) L6 [000431] (1.0 K) + L6 [] (0 B) -> L6 [] (0 B), in 0.0s, output rate 0 B/s
I210812 18:27:25.248516 5753 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n2,pebble,s2] 3282  [JOB 355] sstable deleted 000431
I210812 18:27:25.250822 85 kv/kvserver/replica_raft.go:277  [n1,s1,r42/1:/{Table/46-Max}] 3283  proposing SIMPLE(l3) [(n3,s3):3LEARNER]: after=[(n1,s1):1 (n2,s2):2 (n3,s3):3LEARNER] next=4
I210812 18:27:25.254293 85 kv/kvserver/store_snapshot.go:1138  [n1,replicate,s1,r42/1:/{Table/46-Max}] 3284  streamed INITIAL snapshot e8985187 at applied index 19 to (n3,s3):3LEARNER in 0.00s @ 12 MiB/s: kv pairs: 12, log entries: 0, rate-limit: 8.0 MiB/s, queued: 0.00s
I210812 18:27:25.255020 5769 kv/kvserver/raft_snapshot_queue.go:129  [n1,raftsnapshot,s1,r42/1:/{Table/46-Max}] 3285  skipping snapshot; replica is likely a LEARNER in the process of being added: (n3,s3):3LEARNER
I210812 18:27:25.255233 5757 kv/kvserver/replica_raftstorage.go:824  [n3,s3,r42/3:{-}] 3286  applying snapshot of type INITIAL [id=e8985187 index=19]
I210812 18:27:25.255711 5757 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3287  [JOB 352] ingesting: sstable created 000424
I210812 18:27:25.255777 5757 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3288  [JOB 352] ingesting: sstable created 000429
I210812 18:27:25.255805 5757 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3289  [JOB 352] ingesting: sstable created 000425
I210812 18:27:25.255829 5757 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3290  [JOB 352] ingesting: sstable created 000426
I210812 18:27:25.256644 5757 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3291  [JOB 352] ingesting: sstable created 000427
I210812 18:27:25.256689 5757 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3292  [JOB 352] ingesting: sstable created 000428
I210812 18:27:25.256783 5757 3@vendor/github.com/cockroachdb/pebble/event.go:617  [n3,pebble,s3] 3293  [JOB 353] WAL created 000430 (recycled 000410)
I210812 18:27:25.258155 5692 3@vendor/github.com/cockroachdb/pebble/event.go:585  [n3,pebble,s3] 3294  [JOB 354] flushing 1 memtable to L0
I210812 18:27:25.258242 5692 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3295  [JOB 354] flushing: sstable created 000431
I210812 18:27:25.258672 5692 3@vendor/github.com/cockroachdb/pebble/event.go:589  [n3,pebble,s3] 3296  [JOB 354] flushed 1 memtable to L0 [000431] (9.0 K), in 0.0s, output rate 21 M/s
I210812 18:27:25.258837 5694 3@vendor/github.com/cockroachdb/pebble/event.go:573  [n3,pebble,s3] 3297  [JOB 355] compacting(default) L0 [000431] (9.0 K) + L6 [000423] (196 K)
I210812 18:27:25.259370 5757 3@vendor/github.com/cockroachdb/pebble/event.go:609  [n3,pebble,s3] 3298  [JOB 352] ingested L0:000424 (1.3 K), L0:000429 (1.2 K), L0:000425 (1.6 K), L0:000426 (1.0 K), L0:000427 (1.0 K), L6:000428 (1.0 K)
I210812 18:27:25.259524 5694 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3299  [JOB 355] compacting: sstable created 000432
I210812 18:27:25.259890 5757 kv/kvserver/replica_raftstorage.go:845  [n3,s3,r42/3:/{Table/46-Max}] 3300  applied snapshot of type INITIAL [total=5msingestion=6@4msid=e8985187 index=19]
I210812 18:27:25.260806 85 kv/kvserver/replica_command.go:2118  [n1,replicate,s1,r42/1:/{Table/46-Max}] 3301  change replicas (add [(n3,s3):3] remove []): existing descriptor r42:/{Table/46-Max} [(n1,s1):1, (n2,s2):2, (n3,s3):3LEARNER, next=4, gen=3]
I210812 18:27:25.266914 5694 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n3,pebble,s3] 3302  [JOB 355] compacted(default) L0 [000431] (9.0 K) + L6 [000423] (196 K) -> L6 [000432] (201 K), in 0.0s, output rate 25 M/s
I210812 18:27:25.267786 5580 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3303  [JOB 355] sstable deleted 000423
I210812 18:27:25.267905 5580 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3304  [JOB 355] sstable deleted 000431
I210812 18:27:25.267956 5581 3@vendor/github.com/cockroachdb/pebble/event.go:573  [n3,pebble,s3] 3305  [JOB 358] compacting(default) L0 [000424 000429 000425 000426 000427] (6.2 K) + L6 [000432] (201 K)
I210812 18:27:25.268009 85 kv/kvserver/replica_raft.go:277  [n1,s1,r42/1:/{Table/46-Max}] 3306  proposing SIMPLE(v3) [(n3,s3):3]: after=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I210812 18:27:25.268243 5581 3@vendor/github.com/cockroachdb/pebble/event.go:601  [n3,pebble,s3] 3307  [JOB 358] compacting: sstable created 000433
I210812 18:27:25.280426 85 testutils/testcluster/testcluster.go:1157  [-] 3308  WaitForFullReplication took: 3.333532642s
I210812 18:27:25.291714 5581 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n3,pebble,s3] 3309  [JOB 358] compacted(default) L0 [000424 000429 000425 000426 000427] (6.2 K) + L6 [000432] (201 K) -> L6 [000433] (201 K), in 0.0s, output rate 8.3 M/s
I210812 18:27:25.292443 5695 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3310  [JOB 358] sstable deleted 000424
I210812 18:27:25.292545 5696 3@vendor/github.com/cockroachdb/pebble/event.go:573  [n3,pebble,s3] 3311  [JOB 359] compacting(elision-only) L6 [000428] (1.0 K) + L6 [] (0 B)
I210812 18:27:25.293022 5696 3@vendor/github.com/cockroachdb/pebble/event.go:577  [n3,pebble,s3] 3312  [JOB 359] compacted(elision-only) L6 [000428] (1.0 K) + L6 [] (0 B) -> L6 [] (0 B), in 0.0s, output rate 0 B/s
I210812 18:27:25.293232 5732 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3313  [JOB 359] sstable deleted 000428
I210812 18:27:25.293341 5695 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3314  [JOB 358] sstable deleted 000425
I210812 18:27:25.293696 5695 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3315  [JOB 358] sstable deleted 000426
I210812 18:27:25.293742 5695 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3316  [JOB 358] sstable deleted 000427
I210812 18:27:25.293766 5695 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3317  [JOB 358] sstable deleted 000429
I210812 18:27:25.293787 5695 3@vendor/github.com/cockroachdb/pebble/event.go:605  [n3,pebble,s3] 3318  [JOB 358] sstable deleted 000432
=== CONT  TestStreamIngestionFrontierProcessor
    stream_ingestion_frontier_processor_test.go:242: Leaked goroutine: goroutine 5864 [chan send]:
        github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest.(*mockStreamClient).ConsumePartition.func1(0xc003c56d80, 0xc003c56de0, 0xc001e4e450, 0x1, 0x1, 0x5980390, 0xc005301340, 0xc003555d70, 0x451c7be, 0x12)
        	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest/stream_ingestion_processor_test.go:95 +0x26d
        created by github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest.(*mockStreamClient).ConsumePartition
        	/go/src/github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamingest/stream_ingestion_processor_test.go:87 +0x165
--- FAIL: TestStreamIngestionFrontierProcessor (9.29s)
```
<details><summary>Reproduce</summary>
<p>
To reproduce, try:

```bash
make stressrace TESTS=TestStreamIngestionFrontierProcessor PKG=./pkg/ccl/streamingccl/streamingest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
Parameters in this failure:

- GOFLAGS=-json
</p>
</details>
/cc @cockroachdb/bulk-io adityamaru
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamIngestionFrontierProcessor.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor_test.go
+++ b/pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor_test.go
@@ -25,7 +25,6 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/distsqlutils"
-	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/testcluster"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
@@ -37,7 +36,6 @@ type partitionToEvent map[streamingccl.PartitionAddress][]streamingccl.Event
 
 func TestStreamIngestionFrontierProcessor(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	skip.WithIssue(t, 68795, "flaky test")
 	ctx := context.Background()
 
 	tc := testcluster.StartTestCluster(t, 3 /* nodes */, base.TestClusterArgs{})

